mvpa2/tests/test_testing.py

```diff
@@ -100,4 +114,75 @@ def test_generate_testing_fmri_dataset(tempfile):
     assert_equal(tempfile, filename)
     assert_true(exists(tempfile))
     ds_reloaded = h5load(tempfile)
-    assert_datasets_equal(ds, ds_reloaded, ignore_a={'wtf'})
\ No newline at end of file
+    assert_datasets_equal(ds, ds_reloaded, ignore_a={'wtf'})
+
+
+
+@sweepargs(attribute=['samples', 'sa', 'fa', 'a'])
+@sweepargs(digits=[None, 1, 2, 3])
+def test_assert_datasets_almost_equal(digits, attribute):
+    samples = np.random.standard_normal((2, 5))
+    args = dict(sa=dict(targets=np.asarray([1., 2])),
+                fa=dict(ids=np.asarray([0., 1, 2, 3, 4])),
+                a=dict(a_value=[66]))
+
+    ds = Dataset(samples=samples, **args)
+
+
+    def negate_assert(f):
+        def raiser(*args, **kwargs):
+            assert_raises(AssertionError, f, *args, **kwargs)
+
+
+        return raiser
```

Comment:
screw pycharm - this is just insane... within 5 lines, 1 line of code and 4 of empty spaces -- what good does it do?